js/client.js

```diff
@@ -2,15 +2,15 @@
 // websocket server that's defined in server.js.
 class Client {
   // game is a GameState and should start at the beginning of the game.
-  constructor(name, game) {
-    this.name = name
+  constructor(game) {
     this.game = game
+    this.name = `Guest ${Math.floor(Math.random() * 100)}`
```

Comment:
hmm, did you revert this accidentally? name creation is supposed to happen outside of the client so that we can order operations like:

1/ name is created
2/ game is created
3/ client is created